lib/stitches/generator_files/spec/features/api_spec.rb

```diff
@@ -80,7 +80,7 @@
       correct_code && correct_header
     end
 
-    failure_message_for_should do
+    failure_message do
```

Comment:
Sorry, I was just looking for the dependencies in the gemfile.lock. 

Anyway, the existing line `add_runtime_dependency("rspec-rails", "~> 3")` guarantees that we get rspec 3 or above.